data/external/cats.yml

```diff
@@ -0,0 +1,8 @@
+prepared: ['prepared soups salads', 'instant foods', 'packaged meat', 'prepared meals', 'packaged cheese', 'popcorn jerky', 'refrigerated', 'packaged produce', 'frozen meat seafood', 'frozen meals', 'packaged poultry', 'frozen breakfast', 'canned meals beans', 'refrigerated pudding desserts', 'frozen pizza', 'canned jarred vegetables', 'canned meat seafood', 'lunch meat', 'canned fruit applesauce', 'frozen produce', 'packaged vegetables fruits']
```

Comment:
Would it be useful, since you have multiple ymls, to include a short comment at the top regarding usage.

These are also all very long lists, was there a reason why you hard coded them? How did you get to such long lists?

Reply:
These lists were curated by hand - looked at all aisles in dataset and classified each to one or more (or none) of the categories. I will document this in a `config/README.md`